src/jdk.jpackage/share/classes/jdk/jpackage/internal/resources/HelpResources.properties

```diff
@@ -185,7 +185,7 @@ Generic Options:\n\
 \          Icons, template files, and other resources of jpackage can be\n\
 \          over-ridden by adding replacement resources to this directory.\n\
 \          (absolute path or relative to the current directory)\n\
-\  --runtime-image <file-path>\n\
+\  --runtime-image <file path>\n\
```

Comment:
This seems unrelated to this change.

Reply:
yes - it is unrelated, other than I was adding a new help text with the same syntax (--mac-entitlements <file path>) and was looking for consistent way to say "<file path>" and found that in all cases except this one we said <file path> not <file-path> so I fixed this one too to make them all consistent.